code/obj/item/toys.dm

```diff
@@ -38,6 +39,13 @@
 			else
 				playsound(U, pick(src.sound_attackF1, src.sound_attackF2), 100, 0, 0, U.get_age_pitch())
 
+
+/obj/item/toy/sword/proc/can_reflect_toy()
+	return TRUE
```

Comment:
nitpick but might as well just use `TRUE` as the component arg instead of using a proc ref, since it's always going to be true